src/styles/themes/Theme.type.ts

```diff
@@ -43,4 +43,8 @@ export type Theme = {
   animation: {
     mapMovementTransitionSpeed: string
   }
+  strings: {
```

Comment:
maybe copy is a better term?

Reply:
"strings" is what I have used in other projects. While this is mostly user-facing copy, it can be used for other theme-specific strings as well if we should ever need them.